prod_requirements.txt

```diff
@@ -0,0 +1,112 @@
+affine==2.3.0
+alembic==1.4.2
+appdirs==1.4.3
+asn1crypto==1.4.0
+attrs==19.3.0
+awscli==1.18.88
+azure-core==1.8.0
+azure-storage-blob==12.3.2
+blinker==1.4
+botocore==1.17.11
+brotlipy==0.7.0
+certifi==2020.12.5
+cffi==1.14.0
+chardet==3.0.4
+Click==7.0
+click-plugins==1.1.1
+cligj==0.5.0
+cloudpickle==1.2.2
+colorama==0.4.3
+configparser==5.0.0
+cryptography==2.9.2
+cycler==0.10.0
+cytoolz==0.10.1
+dask==2.10.1
+databricks-cli==0.9.1
+decorator==4.4.1
+docker==4.3.0
+docker-pycreds==0.4.0
+docutils==0.15.2
+efficientnet-pytorch==0.6.3
+entrypoints==0.3
+Fiona==1.8.11
+Flask==1.1.2
+GDAL==3.0.2
+geopandas==0.8.1
+gitdb==4.0.5
+GitPython==3.1.7
+gorilla==0.3.0
+gunicorn==20.0.4
+h5py==2.10.0
+idna==2.9
+imageio==2.6.1
+isodate==0.6.0
+itsdangerous==1.1.0
+Jinja2==2.11.2
+jmespath==0.10.0
+joblib==0.14.1
+kiwisolver==1.1.0
+llvmlite==0.31.0
+Mako==1.1.3
+MarkupSafe==1.1.1
+matplotlib==3.1.2
+mkl-fft==1.1.0
+mkl-random==1.1.0
+mkl-service==2.3.0
+msrest==0.6.18
+munch==2.5.0
+networkx==2.4
+numba==0.48.0
+numpy==1.20.1
+oauthlib==3.0.1
+olefile==0.46
+packaging==20.4
+pandas==1.1.0
+Pillow==7.0.0
+pretrainedmodels==0.7.4
+prometheus-client==0.8.0
+prometheus-flask-exporter==0.15.4
+protobuf==3.12.4
+pyasn1==0.4.8
+pycparser==2.20
+PyJWT==1.7.1
+pynvml==8.0.4
+pyOpenSSL==19.1.0
+pyparsing==2.4.6
+pyproj==2.4.2.post1
+PySocks==1.7.1
+python-dateutil==2.8.1
+python-editor==1.0.4
+pytz==2020.1
+PyWavelets==1.1.1
+PyYAML==5.1.2
+querystring-parser==1.2.4
+rasterio==1.1.1
+requests==2.24.0
+requests-oauthlib==1.3.0
+rsa==3.4.2
+Rtree==0.9.4
+ruamel-yaml==0.15.87
+s3transfer==0.3.3
+scikit-image==0.16.2
+scikit-learn==0.23.2
+scipy==1.5.2
+segmentation-models-pytorch==0.1.2
+Shapely==1.6.4.post2
+six==1.14.0
+smmap==3.0.4
+snuggs==1.4.7
+SQLAlchemy==1.3.13
+sqlparse==0.3.1
+tabulate==0.8.7
+threadpoolctl==2.1.0
+timm==0.2.1
+toolz==0.10.0
+torch==1.4.0
+torchvision==0.5.0
+tornado==6.0.3
+tqdm==4.42.0
+ttach==0.0.3
```

Comment:
is this new dependency added to .travis.yml?

Reply:
not yet, I will add in the next commit